### PR TITLE
修复ChatGPTNextWeb页面的无障碍问题

### PR DESCRIPTION
**Pull request by mayfwl**
#### 💻 变更类型 | Change Type

- [ ] feat    
- [ ] fix    
- [ ] refactor    
- [ ] perf    
- [ ] style    
- [ ] test    
- [ ] docs    
- [ ] ci    
- [x] chore    
- [ ] build    

#### 📝 补充信息 | Additional Information
#5079 

## Summary by CodeRabbit

- **New Features**
  - Enhanced accessibility for the IconButton components, improving screen reader support with new ARIA attributes.
  - Added localization support for the FullScreen action and ShowPassword option in both Chinese and English, enhancing the user experience for speakers of those languages.
  - Introduced new ARIA attributes to various input components, improving usability for individuals relying on assistive technologies.

- **Bug Fixes**
  - Improved clarity and usability of the chat interface through updated title and ARIA attributes for various buttons.

### CODE REVIEW

File: app/locales/cn.ts [hunk 1]
```diff
@@ -42,6 +42,7 @@ const cn = {
       PinToastAction: "查看",
       Delete: "删除",
       Edit: "编辑",
+      FullScreen: "全屏",
```
Comment (Dogtiti) [inline app/locales/cn.ts:45]: en有添加吗？

**Comment by Dogtiti** [comment]
感觉是不是还有设置里面的需要添加一下？检测一下所有界面要素

**Review by Dogtiti — CHANGES_REQUESTED** [review]

**Comment by mayfwl** [comment]
> 感觉是不是还有设置里面的需要添加一下？检测一下所有界面要素

ok

File: app/locales/cn.ts (hunk 1 above)
Comment (mayfwl) [reply app/locales/cn.ts:45]: 没有

**Comment by mayfwl** [comment]
所有的设置添加了

File: app/components/mask.tsx [hunk 2]
```diff
@@ -127,6 +127,8 @@ export function MaskConfig(props: {
             onClose={() => setShowPicker(false)}
           >
             <div
+              tabIndex={0}
```
Comment (Dogtiti) [inline app/components/mask.tsx:130]: 这里的处理是？

File: app/components/ui-lib.tsx [hunk 3]
```diff
@@ -265,16 +265,25 @@ export function Input(props: InputProps) {
...
-export function PasswordInput(props: HTMLProps<HTMLInputElement>) {
-  const [visible, setVisible] = useState(false);
+// 定义一个接口 AriaProps，包含一个可选的 aria 属性
+interface AriaProps {
+  aria?: string;
+}
 
+// 定义一个接口 PasswordInputProps，继承自 HTMLProps<HTMLInputElement> 和 AriaProps
+// 用于描述密码输入框组件的属性
+interface PasswordInputProps extends HTMLProps<HTMLInputElement>, AriaProps {}
```
Comment (Dogtiti) [inline app/components/ui-lib.tsx:275]: type Props = HTMLProps<HTMLInputElement> &  {
  aria?: string;
}
是不是就可以了？

**Review by Dogtiti — CHANGES_REQUESTED** [review]

File: app/components/mask.tsx (hunk 2 above)
Comment (mayfwl) [reply app/components/mask.tsx:130]: 这里头像选择框，因为是div键盘操作的时候不能聚焦，加上tabIndex用键盘tab键操作的时候可以聚焦

File: app/components/ui-lib.tsx (hunk 3 above)
Comment (mayfwl) [reply app/components/ui-lib.tsx:275]: 是的，我修改一下

**Review by Dogtiti — APPROVED** [review]

**Comment by Dogtiti** [comment]
close #5079

**Pull request merged**